README.md

```diff
@@ -67,4 +67,16 @@ Make sure you have an Android emulator running or a device connected and have
 set up your Android development environment:
 https://facebook.github.io/react-native/docs/getting-started.html
 ```
-- In a shell, run: `react-native run-android`
\ No newline at end of file
+- In a shell, run: `react-native run-android`
+
+## Linting the code
+The linting rules were taken from the React Native project itself and can be used by running 
+
+    $ npm run lint
+
+Issues / warning will be shown in the console and most of them can automatically be fixed by running
+
+    $ npm run lint -- --fix
+
+The linting will also be run on each `git push` and fail if there are issues.
```

Comment:
I don't see anything that connects the `prepush` script in package.json to git hooks. 

Typically there's some manual set up step to install git hooks because automation for git hooks is a bit of a security risk.